bdb/bdb_stubs.c

```diff
@@ -52,6 +52,18 @@ void zerob(void* addr,size_t n) {
   memset(addr,0,n);
 }
 
+const void dbt_from_string(DBT *dbt, value v) {
```

Comment:
this introduces new warning : https://travis-ci.org/github/SKS-Keyserver/sks-keyserver/jobs/698818398#L1508

Reply:
Oops, that should have been `static void`, I was thinking about const too much ...